src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts

```diff
@@ -593,6 +595,28 @@ export class InstalledExtensionsView extends ExtensionsListView {
 	}
 }
 
+export class EnabledExtensionsView extends ExtensionsListView {
+
+	async show(query: string): TPromise<IPagedModel<IExtension>> {
+		const queryEnabled = '@enabled';
+		if (!query || ExtensionsListView.isEnabledExtensionsQuery(query)) {
+			return super.show(queryEnabled);
+		}
+		return super.show(queryEnabled + ' ' + query);
```

Comment:
The minute there is a `query` involved, the default `ExtensionsListView` gets used. So the code flow wouldn't reach here. 
Therefore, let's keep it simple and have this as 

```
async show(query: string): TPromise<IPagedModel<IExtension>> {
 		return super.show('@enabled'));		 		
}
```